src/Domain/DomainHelpers.hpp

```diff
@@ -81,6 +85,17 @@ corners_for_rectilinear_domains(const Index<VolumeDim>& domain_extents,
                                 const std::vector<Index<VolumeDim>>&
                                     block_indices_to_exclude = {}) noexcept;
 
+/// \ingroup ComputationalDomainGroup
+/// The number of wedges to include in the Shell domain.
+enum class ShellPieces {
+  /// Use the entire shell
+  All,
+  /// Use only the four equatorial wedges
+  FourOnEquator,
+  /// Use only the lower logical half
+  OneOnEquator
```

Comment:
I don't understand this comment, could you say which physical axis the wedge will be on? Maybe `OneAlongX` would be better